octobot_trading/signals/trading_signal_bundle_builder.py

```diff
@@ -32,6 +32,24 @@ def build(self) -> signals.SignalBundle:
         self._pack_referenced_orders_together()
         return super().build()
 
+    def sort_signals(self):
```

Comment:
I think we could improve this method by using `sort` and a comparator

Reply:
I think it would make it very complex for no reason here as we would need to keep the original order of signals but only "moving" cancel order at the begining of the list.
Or maybe you see a way to do it easily and while being sure to keep initial order ?

Reply:
It could be sorted in 5 lines and without iterating 2 times on the array like https://stackoverflow.com/questions/5213033/sort-a-list-of-lists-with-a-custom-compare-function

Reply:
udpated

Reply:
:+1: